src/Workspaces/Remote/ServiceHub/Services/CodeAnalysisService_NavigateTo.cs

```diff
@@ -43,10 +43,10 @@ internal partial class CodeAnalysisService : IRemoteNavigateToSearchService
 
                     return Convert(result);
                 }
-            }, cancellationToken).ConfigureAwait(false);
+            }, cancellationToken);
         }
 
-        private ImmutableArray<SerializableNavigateToSearchResult> Convert(
+        private IList<SerializableNavigateToSearchResult> Convert(
```

Comment:
:bulb: Generally not a fan of changing return types from immutable data to mutable (increases maintenance cost and complexity)

Reply:
This is a private function that is referenced just from the two functions above that return an IList.